mypyc/ops_misc.py

```diff
@@ -132,3 +132,19 @@ def emit_in(emitter: EmitterInterface, args: List[str], dest: str) -> None:
           error_kind=ERR_FALSE,
           emit=simple_emit('{dest} = PyObject_SetItem({args[0]}, {args[1]}, {args[2]}) >= 0;'),
           priority=0)
+
+py_getattr_op = func_op(
+    name='getattr',
```

Comment:
Should this be `builtins.getattr`? That way explicit `getattr(o, 'attr')` calls would also be optimized?